tests/features/class-onboarding-set-up-shipping.php

```diff
@@ -66,4 +67,36 @@ public function test_zone_does_not_get_created_with_no_default_country() {
 	public function get_empty_base_location() {
 		return null;
 	}
+
+	/**
+	 * Verify that the shipping note does not include label printing for non-us countries.
+	 * Related issue: https://github.com/woocommerce/woocommerce-admin/issues/5394.
+	 */
+	public function test_shipping_note_does_not_include_label_printing_for_non_us_countries() {
+		// Given AU:QLD as the default country.
+		update_option( 'woocommerce_default_country', 'AU:QLD' );
+
+		// When note is retrieved.
+		$note = ReviewShippingSettings::get_note()->get_content();
+
+		// Then it should not contain 'Label printing: enabled'.
+		$contains_string = strpos( $note, 'Label printing: enabled' );
+		$this->assertFalse( $contains_string );
```

Comment:
For future reference, there is an [`assertStringContainsString()`](https://phpunit.readthedocs.io/en/7.5/assertions.html#assertstringcontainsstring) assertion. 😄